src/main/java/study/racing/domain/Car.java

```diff
@@ -3,38 +3,55 @@
 import java.util.Objects;
 
 public class Car {
-    private static final int DEFAULT_POSITION = 0;
-    private static final int MOVE_CONDITION = 4;
-    private int position;
+    private final CarName name;
+    private CarPosition position;
 
-    public Car() {
-        this.position = DEFAULT_POSITION;
+    public Car(String name) {
+        this.name = new CarName(name);
+        this.position = new CarPosition();
     }
 
-    public void move(int randonValue) {
-        if (randonValue >= MOVE_CONDITION) {
-            this.position++;
+    public Car(String name, int position) {
+        this.name = new CarName(name);
+        this.position = new CarPosition(position);
```

Comment:
```suggestion
        this(new CarName(name), new CarPosition(position));
```
주 생성자를 추가하고 주 생성자를 호출해 필드 값을 초기화한다.